### PR TITLE
Added navbar, home page and other routers for the different pages site in reactjs

### LINKED ISSUE #196: Revamp of the complete website for responsiveness
# Describe the bug
The site is not responsive on the different screens

# Screenshots
<p align="center">

</p>
<p align="center">

</p>

# Desktop
 - OS: Ubuntu
 - Browser: Google Chrome
 - Version 20.04

### I want to revamp the complete website using react bootstrap for its responsiveness on the different screens. Currently, in the site at many places images, text and navbar is not responsive.

### PR DESCRIPTION
**Pull request by Shruti3004**
### Description
An idea was pitched to convert the existing code into reactjs for its responsiveness and make it a PWA

Fixes #196 

### Type of Change:

- [x] Code
- [x] Quality Assurance
- [x] User Interface

### How Has This Been Tested?
If you are working on a design then add a gif to show the responsiveness of your design.
Screenshot from 2021-02-02 11-37-26
Screenshot from 2021-02-02 11-37-30
Screenshot from 2021-02-02 11-37-32
Screenshot from 2021-02-02 11-37-35
Screenshot from 2021-02-02 11-37-40
Screenshot from 2021-02-02 11-37-44
<p align="center">

</p>

Please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR) and share the link of deployed site here.
https://anitab-org-reactjs.netlify.app/

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have attached link of deployed site.
- [x] Any dependent changes have been merged

### Code/Quality Assurance Only:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Shruti3004** [comment]
@annabauza Please review!

**Comment by Shruti3004** [comment]
@annabauza please have a look over it!

**Comment by Shruti3004** [comment]
@annabauza please review!

File: reactjs/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (annabauza) [inline reactjs/README.md:1]: Do we need this readme?

**Review by annabauza — CHANGES_REQUESTED** [review]
As we speak. Move it to the top and create wrappers for existing reactjs pages, then use router to display what we have atm. Ones you do that we can start replacing components and pages one by one. What you need to do is make sure router works on static github pages. We tried that before without success.

**Comment by nandini45** [comment]
@Shruti3004  can you make the changes as requested?

**Comment by keshakaneria** [comment]
@Shruti3004 Any updates about the changes requested?

**Comment by Shruti3004** [comment]
@keshakaneria I'll be working on it and will make the pr asap

**Comment by isabelcosta** [comment]
Thank you for your work @Shruti3004 ! Will close due to lack of updates, and make the issue available again.

**Pull request closed without merging**